bin/temporary_reset_script.sh

```diff
@@ -0,0 +1,67 @@
+#COLLATERAL_KEY=
+#COLLATERAL_ADDRESS=
+
+# PLEASE PUT YOUR FAUCET KEY HERE
+FAUCET_PRIVATE_KEY=
+FAUCET_ADDRESS=
+MINING_INTERVAL_IN_SECONDS=30
+
+# PLEASE SET THIS VARIABLES TO YOUR LOCAL DIRECTORIES WITH THE CODE IF YOU WISH TO COMPILE DAPI AND DRIVE
+DAPI_REPO_PATH=/dash/dapi
+DRIVE_REPO_PATH=/dash/drive
+
+BUILD_DAPI_BEFORE_SETUP=false
+BUILD_DAPI_AFTER_SETUP=false
+BUILD_DRIVE=false
+
+CONFIG_NAME="local"
+
+MASTERNODES_COUNT=3
+
+echo "Removing all docker containers and volumes..."
+docker rm -f -v $(docker ps -a -q); docker volume prune -f; rm -rf ~/.dashmate/
+
+./bin/dashmate config:set --config=${CONFIG_NAME} core.debug 1
+
+if [ $BUILD_DRIVE == true ]
+then
+  echo "Setting drive build directory"
+  ./bin/dashmate config:set --config=${CONFIG_NAME} platform.drive.abci.docker.build.path $DRIVE_REPO_PATH
+fi
+
+if [ $BUILD_DAPI_BEFORE_SETUP == true ]
+then
+  echo "Setting dapi build directory before the setup"
+  ./bin/dashmate config:set --config=${CONFIG_NAME} platform.dapi.api.docker.build.path $DAPI_REPO_PATH
+fi
+
+./bin/dashmate setup ${CONFIG_NAME} -v --node-count=${MASTERNODES_COUNT}
+
+echo "Setting log levels to trace"
+./bin/dashmate config:set --config=${CONFIG_NAME} platform.drive.abci.log.stdout.level trace
+
+# Set up a miner for tests
+echo "Enabling miner in config with an interval of ${MINING_INTERVAL_IN_SECONDS} seconds"
+./bin/dashmate config:set --config=${CONFIG_NAME}_seed core.miner.enable true
+./bin/dashmate config:set --config=${CONFIG_NAME}_seed core.miner.interval "${MINING_INTERVAL_IN_SECONDS}s"
+  for (( NODE_INDEX=1; NODE_INDEX<=MASTERNODES_COUNT; NODE_INDEX++ ))
+  do
+      ./bin/dashmate config:set --config=${CONFIG_NAME}_${NODE_INDEX} core.miner.interval "${MINING_INTERVAL_IN_SECONDS}s"
+  done
```

Comment:
We don't need this with https://github.com/dashevo/dashmate/pull/360

Reply:
It's not merged yet though. Do you want to merge #360 first?

Reply:
It's merged. Now we have -d and -m flags so you can remove unnecessary configuration from here.

Reply:
Done